app/onepassword_events_api/appserver/static/javascript/views/setup_page.js

```diff
@@ -41,6 +87,33 @@ export async function perform(splunk_js_sdk, authToken, setup_options) {
     authToken
   );
 
+  // Disable and reload the app to update the config
```

Comment:
Are we sure we need to reload? I was under the impression other apps we reviewed, that were performing similar work to us, didn't need to reload.

Reply:
From my testing, if I don't disable the inputs and reload the app first, the 2 binaries won't actually gets restarted. So this is similar to how we disable/re-enable the 2 scripted inputs in the data inputs page.